HerPublicWebsite/appsettings.json

```diff
@@ -33,5 +33,10 @@
 
     "S3": {
         "BucketName": "TODO dev bucket name here"
+    },
+
+    "SessionCache": {

```

Comment:
I assume we don't need to create a migration to add this table?

Reply:
No - the plugin handles it all internally, although we can disable that behaviour if you'd prefer to have it handled by EF.